README.md

```diff
@@ -20,6 +20,62 @@ Note the address bar indicating google.com. Also, since the content of USA Today
 - Can't request permissions
 - Can't be added to homescreen
 
+## Goals and non-goals
+
+### Goals
+
+* **authenticated-sharing**: Alice can download a package containing
+  a [PWA](https://developers.google.com/web/progressive-web-apps/checklist) from
+  origin `O`, and transmit it to their peer Bob, and then Bob can install
+  the PWA with a proof that it came from `O`.
+* **streamed-loading**: When a package is downloaded over TLS from its original
+  origin, the UA can load the package as it downloads.
+* **random-access**: When a package is stored on disk, the UA can access
+  arbitrary resources without a linear scan.
+* **sub-packages**: A package from origin `A` can contain resources from origin
```

Comment:
The goal's name "sub-packages" refers to our current solution to the real goal which is "multiple origins can make same-origin requests"